File: applications/registration/src/app/dataset/distribution/distribution.ts
import {SkosConcept} from '../skosConcept';
export interface Distribution {
    id?: string;
    uri?: string;
    type?: string;
    title?: {[language: string]: string};
    description?: {[language: string]: string};
    downloadURL?: string[];
    accessURL?: string[];
    format?: string[];
    license?: SkosConcept;
    conformsTo?: SkosConcept[];
    page?: SkosConcept[];
}

File: libraries/shared/src/main/java/no/dcat/shared/Distribution.java
package no.dcat.shared;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Data;
import lombok.ToString;
import no.dcat.shared.SkosConcept;

import java.util.List;
import java.util.Map;

/**
 * Created by bjg on 24.02.2017.
 */
@Data
@ToString(includeFieldNames = false)
@JsonInclude(JsonInclude.Include.NON_NULL)
public class Distribution {
    private String id;
    private String uri;
    private Map<String,String> title;
    private Map<String,String> description;
    private List<String> downloadURL;
    private List<String> accessURL;
    private SkosConcept license;
    private Boolean openLicense;
    private List<SkosConcept> conformsTo;
    private List<SkosConcept> page;
    private List<String> format;
    private String type;
    private DataDistributionService accessService;
}
